### PR TITLE
LTD-5522 fix unexpected search results

### PR DESCRIPTION
**Pull request by markj0hnst0n**
### Aim

Ensure that all hits for search results are presented for Search Products.

[LTD-5522](https://uktrade.atlassian.net/browse/LTD-5522)

[LTD-5522]: https://uktrade.atlassian.net/browse/LTD-5522?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: api/search/product/views.py [hunk 1]
```diff
@@ -102,9 +102,6 @@ def get_queryset(self):
                     "inner_hits": {
                         "size": 200,
                         "name": "related",
-                        "collapse": {
-                            "field": "context",
-                        },
                         "sort": [
```
Comment (markj0hnst0n) [inline api/search/product/views.py:105]: The search results were being 'collapsed' twice and so only the first 'collapse' on product name (first key in this dictionary) was being used.  The second collapse had the effect of hiding the search hits

Comment (depsiatwal) [reply api/search/product/views.py:105]: do you mind linking here the other collapse ?

Comment (markj0hnst0n) [reply api/search/product/views.py:105]: It was removed to enable the first collapse to work correctly.  Originally `get_queryset` looked like this:
image

Comment (markj0hnst0n) [reply api/search/product/views.py:105]: You can see collapse is the first key in the query dict

**Review by depsiatwal — APPROVED** [review]
lgtm

**Review by kevincarrogan — COMMENTED** [review]
Could we get a test around this to make sure we don't accidentally break this again in future?

**Review by kevincarrogan — APPROVED** [review]

**Pull request merged**